daemon-systemd.in

```diff
@@ -10,8 +10,8 @@ PIDFile=@lockfile@
 PermissionsStartOnly=true
 ExecStartPre=-/usr/bin/mkdir /var/run/naemon
 ExecStartPre=/usr/bin/chown -R naemon:naemon /var/run/naemon/
-ExecStartPre=su - naemon -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
-ExecStart=@bindir@/naemon -d @pkgconfdir@/naemon.cfg
+ExecStartPre=su - naemon -s /bin/sh -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
+ExecStart=@bindir@/naemon -ud @pkgconfdir@/naemon.cfg
```

Comment:
that might break some "check_procs" when uses filter for the naemon process, so i'd like to keep it like it is.